ruby-bionic/2.5.3/Dockerfile

```diff
@@ -0,0 +1,103 @@
+FROM socrata/base-bionic
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+# skip installing gem documentation
+RUN mkdir -p /usr/local/etc \
+	&& { \
+		echo 'install: --no-document'; \
+		echo 'update: --no-document'; \
+	} >> /usr/local/etc/gemrc
+
+ENV RUBY_MAJOR 2.5
+ENV RUBY_VERSION 2.5.3
```

Comment:
I'm hopeful we can try to not have a bionic 2.5.3 image at all and just get people to do both the bionic and ruby 2.6 upgrade in one step to keep one less image around and reduce duplicate testing effort.  

That said, if upgrading something to 2.6 really does add that much scope it would be fine to add one.

Reply:
I've removed 2.5.3.

Reply:
After a discussion with Johnny and Roark I'm adding this back but using 2.6.5 will be encouraged.

Reply:
I definitely ran into an issue with heph when updating the gemfile to use 2.6.5.  It was easy to navigate and resolve but that project doesn't have a huge dependency list.